README.adoc

```diff
@@ -64,6 +64,7 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+For a new pull request, it will trigger the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] and if the pull request contains any change related to add-on, it will run the integration tests related to the add-ons.
```

Comment:
@budhrg Somehow the  sentence is confusing. The confusion is  around pull request build job and running integration tests. Here is my suggestion:
`For a new pull request, the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered. But the build job runs the integration tests only when the pull request contains any change related to add-on.` WDYT?